examples/standalone_mode_demo.ipynb

```diff
@@ -0,0 +1,609 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "GSgnnNodeModel(\n",
+       "  (_node_input_encoder): GSNodeEncoderInputLayer(\n",
+       "    (dropout): Dropout(p=0.0, inplace=False)\n",
+       "    (proj_matrix): ParameterDict(\n",
+       "        (movie): Parameter containing: [torch.FloatTensor of size 128x128]\n",
+       "        (user): Parameter containing: [torch.FloatTensor of size 128x128]\n",
+       "    )\n",
+       "    (input_projs): ParameterDict()\n",
+       "    (ngnn_mlp): ModuleDict(\n",
+       "      (movie): NGNNMLP(\n",
+       "        (dropout): Dropout(p=0.0, inplace=False)\n",
+       "        (ngnn_gnn): ParameterList()\n",
+       "      )\n",
+       "      (user): NGNNMLP(\n",
+       "        (dropout): Dropout(p=0.0, inplace=False)\n",
+       "        (ngnn_gnn): ParameterList()\n",
+       "      )\n",
+       "    )\n",
+       "  )\n",
+       "  (_gnn_encoder): RelationalGCNEncoder(\n",
+       "    (_layers): ModuleList(\n",
+       "      (0): RelGraphConvLayer(\n",
+       "        (conv): HeteroGraphConv(\n",
+       "          (mods): ModuleDict(\n",
+       "            (('movie', 'rating-rev', 'user')): GraphConv(in=128, out=128, normalization=right, activation=None)\n",
+       "            (('user', 'rating', 'movie')): GraphConv(in=128, out=128, normalization=right, activation=None)\n",
+       "          )\n",
+       "        )\n",
+       "        (ngnn_mlp): NGNNMLP(\n",
+       "          (dropout): Dropout(p=0.0, inplace=False)\n",
+       "          (ngnn_gnn): ParameterList()\n",
+       "        )\n",
+       "        (dropout): Dropout(p=0.0, inplace=False)\n",
+       "      )\n",
+       "    )\n",
+       "  )\n",
+       "  (_decoder): EntityClassifier(\n",
+       "    (dropout): Dropout(p=0, inplace=False)\n",
+       "  )\n",
+       "  (_loss_fn): ClassifyLossFunc(\n",
+       "    (loss_fn): CrossEntropyLoss()\n",
+       "  )\n",
+       ")"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# create a built-in gsf model\n",
+    "model = gs.create_builtin_node_gnn_model(\n",
+    "    train_data.g, \n",
+    "    config, \n",
+    "    train_task=True)\n",
+    "model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of model parameters: 84480\n",
+      "Number of trainable parameters: 84480\n"
+     ]
+    }
+   ],
+   "source": [
+    "num_params = sum(p.numel() for p in model.parameters())\n",
+    "num_trainable_params = sum(p.numel() for p in model.parameters() if p.requires_grad)\n",
```

Comment:
Because the results of the two variables are the same, which is 84480, readers may ask what is the difference between the two variables. Do we have an example having different values for the two variables?